src/components/list/list-base.scss

```diff
@@ -99,6 +99,8 @@ $md-list-item-subheader-height: 6 * $baseline-grid !default;
     margin-top: $baseline-grid;
     margin-bottom: $baseline-grid;
     border-radius: 50%;
+
+    flex-shrink: 0; // Per https://bugs.webkit.org/show_bug.cgi?id=146020
```

Comment:
It's resolved in the latest Webkit version, but it's not resolved in the latest Safari version (probably in the Technology Preview).

I tested that on my Mac Book Pro and the bug is still valid.